src/displayapp/screens/WatchFaceNumerals.cpp

```diff
@@ -0,0 +1,155 @@
+#include "displayapp/screens/WatchFaceNumerals.h"
+
+#include <lvgl/lvgl.h>
+#include <cstdio>
+#include "displayapp/screens/Symbols.h"
+#include "components/ble/NotificationManager.h"
+#include "components/settings/Settings.h"
+
+using namespace Pinetime::Applications::Screens;
+
+WatchFaceNumerals::WatchFaceNumerals(Controllers::DateTime& dateTimeController,
+                                     Controllers::NotificationManager& notificationManager,
+                                     Controllers::Settings& settingsController,
+                                     Controllers::FS& filesystem)
+  : currentDateTime {{}},
+    dateTimeController {dateTimeController},
+    notificationManager {notificationManager},
+    settingsController {settingsController} {
+
+  lfs_file f = {};
+  if (filesystem.FileOpen(&f, "/fonts/rounded_large.bin", LFS_O_RDONLY) >= 0) {
+    filesystem.FileClose(&f);
+    font_large = lv_font_load("F:/fonts/rounded_large.bin");
+  }
+
+  if (filesystem.FileOpen(&f, "/fonts/rounded_small.bin", LFS_O_RDONLY) >= 0) {
+    filesystem.FileClose(&f);
+    font_small = lv_font_load("F:/fonts/rounded_small.bin");
+  }
+
+  notificationIcon = lv_obj_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_bg_color(notificationIcon, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_AQUA);
+  lv_obj_set_style_local_radius(notificationIcon, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_RADIUS_CIRCLE);
+  lv_obj_set_size(notificationIcon, 16, 16);
+  lv_obj_align(notificationIcon, lv_scr_act(), LV_ALIGN_IN_BOTTOM_LEFT, 4, -76);
+  lv_obj_set_hidden(notificationIcon, true);
+
+  labelTimeHour = lv_label_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_text_font(labelTimeHour, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, font_large);
+  lv_obj_set_style_local_text_color(labelTimeHour, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_GRAY);
+  lv_obj_align(labelTimeHour, lv_scr_act(), LV_ALIGN_IN_BOTTOM_RIGHT, -160, -125);
+
+  labelTimeMinute = lv_label_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_text_font(labelTimeMinute, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, font_large);
+  lv_obj_set_style_local_text_color(labelTimeMinute, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_AQUA);
+  lv_obj_align(labelTimeMinute, lv_scr_act(), LV_ALIGN_IN_BOTTOM_RIGHT, -160, 0);
+
+  labelTimeAMPM1 = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(labelTimeAMPM1, "");
+  lv_obj_set_style_local_text_font(labelTimeAMPM1, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, font_small);
+  lv_obj_set_style_local_text_color(labelTimeAMPM1, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(0x999999));
```

Comment:
Since this colour is being used a couple of times I think it would be best if were a `static constexpr` member of the class (i.e `static constexpr lv_color_t AMPMColor = lv_color_hex(0x999999)` or similar)